WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -53,6 +53,7 @@ + (NSArray *)routes
       [NSString stringWithFormat:@"%@ is not a valid URL", url]
     );
   }
+  #pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
wouldn't it be more correct to replace the deprecated declaration with the up-to-date one?(openURL:options:completionHandler:)